src/App.js

```diff
@@ -1,30 +1,20 @@
-// src/App.js
-import React from 'react';
-import {
-  BrowserRouter as Router, Routes, Route, Link,
-} from 'react-router-dom';
-import HomePage from './components/HomePage';
-import CategoriesPage from './components/CategoriesPage';
+import { Route, Routes } from 'react-router-dom';
+/* eslint-disable */
+import HomePage from './components/homePage';
+import CategoriesPage from './components/categoriesPage';
```

Comment:
- [ ] Please fix the path of your files because when I run your project it gives me the following errors so kindly fix it and also check below :point_down:  my suggestion as well :slightly_smiling_face: 

![Screenshot from 2023-08-06 01-25-56](https://github.com/Sonikak004/book-store/assets/45226848/01b79af8-26fe-4ed4-a812-8804364b1c38)

```suggestion
import HomePage from './components/HomePage';
import CategoriesPage from './components/CategoriesPage';
```